### ISSUE TITLE
I cannot enter the game in Windows XP

### ISSUE THREAD

**Issue by V1RTU4LGD** [ExOptsTeam/NFSU2ExOpts#17]
Hi!! I have a problem wanting to install NFSU2ExtraOptions, I have a very old computer (it has Windows XP). When I pass the files of the NFSU2ExOpts.v4.0.0.1337 and enter the game I get an error that says "The entry point of the SHGetKnowFolderPath procedure cannot be found in the dynamic links library SHELL32.dll.". I would like to know if somebody have any solution for that problem.

**Comment by ThatD07** [comment]
ExOpts doesn't support XP, since it's an obsolete OS

**Comment by V1RTU4LGD** [comment]
So it only works in versions before 4.0, right?

**Comment by mirh** [comment]
Nope, even the first code drop available here didn't support it already. 
On the other hand you can try to compile yourself this with the `v141_xp` toolset, and see what comes out of it.
